invisible_cities/io/mcinfo_io.py

```diff
@@ -330,32 +456,68 @@ def read_mcparticles_df(h5in    : tb.file.File,
     return parts
 
 
-def load_mcsensor_response(file_name: str,
-                           event_range=(0, int(1e9))) -> Mapping[int, Mapping[int, Waveform]]:
-
-    with tb.open_file(file_name, mode='r') as h5in:
-        return read_mcsns_response(h5in, event_range)
-
-
-def get_sensor_binning(file_name : str) -> Tuple:
+def get_sensor_binning(file_name : str) -> pd.DataFrame:
     """
     Looks in the configuration table of the
     input file and extracts the binning used
     for both types of sensitive detector.
     """
-    config   = pd.read_hdf(file_name, 'MC/configuration')
-    bins     = config[config.param_key.str.contains('time_binning')]
-    pmt_bin  = bins.param_value[bins.param_key.str.contains('Pmt')].iloc[0]
-    pmt_bin  = float(pmt_bin.split()[0]) * units_dict[pmt_bin.split()[1]]
-    sipm_bin = bins.param_value[bins.param_key.str.contains('SiPM')].iloc[0]
-    sipm_bin = float(sipm_bin.split()[0]) * units_dict[sipm_bin.split()[1]]
+    if is_oldformat_file(file_name):
+        return sensor_binning_old(file_name)
+    else:
+        return sensor_binning_new(file_name)
+
+
+def sensor_binning_old(file_name : str) -> pd.DataFrame:
+    # Is a pre-Feb 2020 MC file
+    config = pd.read_hdf(file_name, 'MC/configuration').set_index('param_key')
```

Comment:
I think is better to use `dst_io.load_dst` ( that opens file using pytables) since we already saw that in different pandas version this might not work, depending on how file was created (ie pytables or pandas writer)

Reply:
Do you have an example of a fail? I've never seen it fail in this context due to pandas versions.

Reply:
The beersheba_tests fail due to pandas not being able to read PSF file in version 0.25 (we use 0.23). This is an issue that needs to be solved but obviously not related to this PR